ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/smartTree/TreeStyles.java

```diff
@@ -36,6 +36,8 @@
 
         String infoTextContainer();
 
+        String vcsHeadLinkContainer();
```

Comment:
It would be better to move this style declaration to general resource bundle `org.eclipse.che.ide.Resources.CoreCss` and make it accessible for every component which wants to colorize standard html link. WDYT?